module-more-apps/src/main/java/com/orange/ods/module/moreapps/domain/MoreAppsEntities.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Software Name: Orange Design System
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ods.module.moreapps.domain
+
+import android.util.DisplayMetrics
+
+internal interface MoreAppsItem
+
+internal data class AppsSection(
+    val name: String?,
+    val items: List<MoreAppsItem>
+) : MoreAppsItem
+
+internal data class AppsList(
+    val items: List<MoreAppsItem>
+) : MoreAppsItem
```

Comment:
What do we do if we receive an AppSection with a null name from the backend?

Reply:
I keep only the `AppsList` class with a nullable `name`. If the name is null it will display it as a simple list, otherwise it will display it as a section.